api/proto/teleport/integration/v1/awsoidc_service.proto

```diff
@@ -53,3 +56,44 @@ message ListDatabasesResponse {
   // If non-empty, it can be used to request the next page.
   string next_token = 2;
 }
+
+// DeployDatabaseServiceRequest is a request to deploy .
+message DeployDatabaseServiceRequest {
+  // Integration is the AWS OIDC Integration name.
+  string integration = 1;
+  // Region is the AWS Region
+  string region = 2;
+  // TaskRoleARN is the AWS IAM Role received by the deployed service.
+  string task_role_arn = 3;
+  // TeleportVersion is the teleport version to be deployed.
+  // This is used to fetch the correct tag for the teleport container image.
+  // Eg, 14.3.4 (no "v" prefix)
+  string teleport_version = 4;
+  // DeploymentJoinTokenName is the Teleport IAM Join Token to be used by the deployed
+  // service to join the cluster.
+  string deployment_join_token_name = 5;
+  // Deployments is a list of services that will be deployed.
+  repeated DeployDatabaseServiceDeployment deployments = 6;
+}
+
+// DeployDatabaseServiceDeployment represents a single deployment.
+message DeployDatabaseServiceDeployment {
+  // TeleportConfigString is the teleport.yaml configuration (base64 encoded) used by teleport.
```

Comment:
This is the format expected by `teleport start --config-string <TeleportConfigString>`.

Reply:
Yeah I saw it later on. It's a weird choice, as an RPC has no problems sending any string over (unlike a literal typed on the shell) and it will make the string bigger.

Reply:
Yeah, we could send it as a string representing the yaml of the configuration and then b64-encode it in the Auth Server, but I was trying to not change logic too much with this refactor..